docs/src/modules/components/ApiPage.js

```diff
@@ -190,6 +193,41 @@ export default function ApiPage(props) {
     >
       <MarkdownElement>
         <h1>{pageContent.name} API</h1>
+        {deprecated ? (
```

Comment:
I tried that, but I ended up overriding font size, icon alignment, etc. to make it look like the success alert we display under the "Demos" heading below.

|current|ApiWarning|
|---|---|
|<img width="848" alt="Screenshot 2024-05-24 at 17 40 00" src="https://github.com/mui/material-ui/assets/7225802/7b1d0e44-32af-4a61-be2b-2d553e71813a">|<img width="854" alt="Screenshot 2024-05-24 at 17 40 31" src="https://github.com/mui/material-ui/assets/7225802/b953f75c-96ac-4f94-a3ce-facf7146851b">|

Looks especially different if the text is longer since the icon is vertically centered:

<img width="640" alt="Screenshot 2024-05-24 at 17 42 47" src="https://github.com/mui/material-ui/assets/7225802/36bf1203-3697-46e3-bc33-6f79e468d259">

Maybe I can add a couple of props to `ApiWarning` to be able to display a different font size and have a different vertical alignment. Is this what you had in mind?

---

BTW, we also have different implementations for alerts/callouts in markdown-generated files (e.g. `:::error`, `:::info`, etc.).

Reply:
It was just to be sure you did not miss the existence of the component

Reply:
Created this PR here in light of this thread → https://github.com/mui/material-ui/pull/42386. It tackles the alert design standardization 🤙

Reply:
Thanks @danilo-leal! I'll update this PR once yours is merged.

Reply:
Just merged it 🤙